docs/en/04_Changelogs/4.0.0.md

```diff
@@ -828,11 +828,21 @@ that previously visible assets remain visible to the public site.
 If additional security or visibility rules should be applied to File dataobjects, then
 make sure to correctly extend `canView` via extensions.
 
+Note that any file which is not in the `File.allowed_extensions` config will be deleted during migration.
```

Comment:
"Deleted" or just "DataObject isn't created for it and file on filesystem will still remain"

I guess I'd be a bit more careful with wording here, I know there's a sentence below, maybe bring them together to reduce confusion